submission/server/main.go

```diff
@@ -60,5 +60,6 @@ func main() {
 	s := submission.NewProxyServer(*logListPath, submission.GetDistributorBuilder(plc, lcb, mf), *addPreChainTimeout)
 	s.Run(*logListRefreshInterval, *rootsRefreshInterval)
 	http.Handle("ct/v1/proxy/add-pre-chain/", s.HandleAddPreChain())
+	http.Handle("/", s.HandleInfo())
```

Comment:
This is a very odd pattern (this, and the line above)? Normally, you pass in a function directly to `http.Handle`, but those return closures?

What I would expect would be that `s.HandleAddPrechain` and `s.HandleInfo` would themselves match the signature of `http.HandlerFunc` (instead of *returning* a function matching it), and there wouldn't be parentheses here (as in `http.Handle("/", s.HandleInfo)`).

Any reason why it's not like that?

Reply:
Had some remote ideas on testing. Moving to simple schema for now.